packages/fury-adapter-oas3-parser/lib/parser/oas/parsePathItemObject.js

```diff
@@ -127,6 +128,7 @@ function parsePathItemObject(context, member) {
     [hasKey('summary'), parseString(context, name, false)],
     [hasKey('description'), parseCopy(context, name, false)],
     [hasKey('parameters'), R.curry(parseParameters)(context, member.key)],
+    [hasKey('servers'), R.compose(parseServersArray(context, name), getValue)],
```

Comment:
There's a comment `// FIXME Parse servers` a few lines below, could you remove that please.

Reply:
removed